### PR TITLE
pdns: update to 4.8.3

### PR DESCRIPTION
**Pull request by Habbie**
Maintainer: me
Compile tested: github CI
Run tested: Docker openwrt/rootfs x86_64

Description:

### CODE REVIEW

File: net/pdns/files/pdns.conf-dist [hunk 1]
```diff
@@ -140,6 +140,11 @@
 #
 # default-api-rectify=yes
 
+#################################
+# default-catalog-zone	Catalog zone to assign newly created primary zones (via the API) to
+#
+# default-catalog-zone=
```
Comment (BKPepe) [inline net/pdns/files/pdns.conf-dist:146]: There isn't said anything about this in the commit description. You should do these changes in separate commit.

**Review by BKPepe — CHANGES_REQUESTED** [review]

**Comment by 1715173329** [comment]
ping @Habbie

File: net/pdns/files/pdns.conf-dist (hunk 1 above)
Comment (Habbie) [reply net/pdns/files/pdns.conf-dist:146]: I can update the commit description, but doing it in a separate commit doesn't make sense. It's part of the version bump.

Comment (Habbie) [reply net/pdns/files/pdns.conf-dist:146]: commit description updated
